client/store/modules/totalOodlet.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Created by toma on 09.11.16..
+ */
+import array from 'lodash/array';
+import object from 'lodash/object';
+
+function thingyAdd (state, quantifiedThingy) {
+  let index = array.findIndex(state.quantifiedThingies, {id: quantifiedThingy.id});
+  if (index > -1) {
+    state.quantifiedThingies[index].qty += quantifiedThingy.qty;
+  }
+  else {
+    state.quantifiedThingies.push({} = object.merge({}, quantifiedThingy));
+  }
+}
+
+function thingyRemove (state, quantifiedThingy) {
+  let index = array.findIndex(state.quantifiedThingies, {id: quantifiedThingy.id});
+  if (index > -1) {
+    state.quantifiedThingies[index].qty -= quantifiedThingy.qty;
+    if(state.quantifiedThingies[index].qty <= 0) {
+      array.remove(state.quantifiedThingies, {id: quantifiedThingy.id});
+    }
+  }
+}
+
+function addOodlet(state, pendingOodlet) {
+  if (array.indexOf(state.oodletIds, pendingOodlet.id) < 0) {
+    state.oodletIds.push(pendingOodlet.id);
+    return true;
+  }
+  return false;
+}
+
+function removeOoodlet(state, pendingOodlet) {
+  if (array.indexOf(state.oodletIds, pendingOodlet.id) > -1) {
+    array.pull(state.oodletIds, pendingOodlet.id);
+    return true;
+  }
+  else {
+    return false;
+  }
+}
+
+const state = {
+  id: '',
+  quantifiedThingies: [],
+  oodletIds: []
+};
+
+const mutations = {
+  totalOodletSet(state, totalOodlet) {
+    state.id = totalOodlet.id;
+    state.quantifiedThingies = (() => { return totalOodlet.quantifiedThingies ? totalOodlet.quantifiedThingies : [];})();
+    state.oodletIds = (() => { return totalOodlet.oodletIds ? totalOodlet.oodletIds : [];})();
+  },
+  
+  oodletAdd(state, pendingOodlet) {
+    if(addOodlet(state, pendingOodlet)) {
```

Comment:
Consider renaming `addOodlet` to something else. It reads very confusing this way.